docs/langref/src/intro.md

```diff
@@ -1,16 +1,17 @@
-# The `.slint` language reference
+# The `.slint` Language Reference
 
-The Slint design markup language is used to describe graphical user interfaces:
+The Slint design markup language describes graphical user interfaces in a way
+easy to learn, use and extend.
+
+It allows to:
```

Comment:
I think we should not talk about "allowance" - that implies a permission model that doesn't exist. "Use it to..." or "Declare XYZ with" is better, but in this case I think the previous list is sufficient without an intro.

```suggestion
```